cross/tryrun.cmake

```diff
@@ -3,6 +3,7 @@ set(TARGET_ARCH_NAME $ENV{TARGET_BUILD_ARCH})
 
 macro(set_cache_value)
   set(${ARGV0} ${ARGV1} CACHE STRING "Result from TRY_RUN" FORCE)
+  set(${ARGV0}__TRYRUN_OUTPUT ${ARGV1} CACHE STRING "Output from TRY_RUN" FORCE)
```

Comment:
According to cmake docs, this TRYRUN_OUTPUT is supposed to contain the stdout / stderr output of the related test when running on the target platform. Here you set it to the exit code. But I guess that it doesn't matter what we set it to as long as cmake doesn't care.